### PR TITLE
Quick fix for #11052

### PR DESCRIPTION
**Pull request by CyaNinix**
Quick fix for the issue @hultonha found on the PR #11052.

Please note I couldn't test it and won't have the time to do so until thuesday, but it should be good

### CODE REVIEW

File: Code/Editor/EditorViewportWidget.cpp [hunk 1]
```diff
@@ -947,14 +947,21 @@ void EditorViewportWidget::SetViewportId(int id)
...
         {
             OnDefaultCameraNearFarChanged();
         }
     );
-    m_editorViewportSettingsCallbacks->SetFarPlaneDistanceChangedEvent(m_nearFarPlaneDistanceHandler);
-    m_editorViewportSettingsCallbacks->SetNearPlaneDistanceChangedEvent(m_nearFarPlaneDistanceHandler);
+    m_editorViewportSettingsCallbacks->SetNearPlaneDistanceChangedEvent(m_nearPlaneDistanceHandler);
+
+    m_farPlaneDistanceHandler = SandboxEditor::NearFarPlaneChangedEvent::Handler(
+        [this](float /*farPlaneDistance*/)
```
Comment (hultonha) [inline Code/Editor/EditorViewportWidget.cpp:959]: uber nit: While we're here would you be able to update this to 

```
[[maybe_unused]] float farPlaneDistance
...
[[maybe_unused]] float nearPlaneDistance
```

Just so it's easier to debug in future to you can see what the value of near/farPlaneDistance

**Review by hultonha — APPROVED** [review]
Thanks for fixing this so quickly! 😄

**Review by cgalvan — APPROVED** [review]

**Review by AMZN-daimini — APPROVED** [review]

**Review by lsemp3d — APPROVED** [review]

File: Code/Editor/EditorViewportWidget.cpp (hunk 1 above)
Comment (CyaNinix) [reply Code/Editor/EditorViewportWidget.cpp]: Here you are, sorry for the late update.

Comment (hultonha) [reply Code/Editor/EditorViewportWidget.cpp]: Amazing thank you! 🙂

**Comment by hultonha** [comment]
Thanks @lsemp3d and thanks again @CyaNinix 👍

**Pull request merged**